### PR TITLE
[Keymap] Config tweaks and adding zeal60 RGB files to userspace

### PR DESCRIPTION
**Pull request by stanrc85**
## Description

- Tweaked tapping_term value
- Add zeal60 RGB backlight files to userspace since I've made small changes to them.  Without access to zeal60 config tool on HS60 keyboard I had to hard code in some values for RGB layer indicators. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by Wilba6582** [comment]
Can you explain the need to copy and change the `rgb_backlight.c` file?

I am not sure if copying QMK source files into `users` is the best approach. If you want to override default behaviour in a "common" source file in QMK, then why not just do that in your own fork, i.e. maintain modifications to the original source file and merge as required? I don't see the need to push in wholesale copies of source files into `users` and thus use QMK master repo instead.

If the change to `rgb_backlight.c` is of use to others, then why not propose it to me to change the original code?

This is just my opinion - I think QMK repo is already bloated with lots of user layouts that are only of interest to the user that committed them, and this seems to be an extension of that. Obviously I have no problem with you using that code to customize your keyboard to your needs, this is only a criticism of how that should be best managed.

**Comment by stanrc85** [comment]
Someone please correct me if I'm wrong because I'm figuring this all out as I go, but from the best I can tell the only way to modify variables in the zeal60 RGB file is using a special built zeal60.exe (https://github.com/Wilba6582/zeal60/blob/master/readme.txt) which doesn't exist or work for the HS60/v2, so without that I decided to hard code in some changes to suit my HS60/v2 which uses the zeal60 rgb files. I had been keeping the modified version locally but I've been contacted by a few other HS60/v2 owners about how I did it so I was posting it to share.  I agree its far from ideal to be duplicating whole files like this.  If I was a better programmer I would figure out how to make some of those options user configurable like TAPING_TERM or something. 

I'm happy to go back to keeping it locally until I can find a cleaner way to modify the rgb_backlight.c file to allow user controlled values.

**Comment by Wilba6582** [comment]
@stanrc85 aaah OK this is entirely my fault, those settings have not been added to VIA Configurator yet, and the zeal60.exe tool is still the only way to configure it. Perhaps as a short term solution, I can make zeal60.exe work with HS60/v2 so you can change these settings, while we all wait for VIA Configurator to catch up.

**Comment by stanrc85** [comment]
How hard would it be to make the caps lock and layer indicators user configurable variables? Would it make it harder to implement with via down the road? If you don't mind I might work on that. I like the simplicity of being able to do it all in QMK personally.

**Comment by Wilba6582** [comment]
That would be easy, it's just not particularly user-friendly because specifying the key/keys to light up is a bit obscure.

Add this to every `keyboards/.../config.h` that uses `rgb_backlight.c`:

`#define RGB_BACKLIGHT_CAPS_LOCK_INDICATOR { .color = { .h = 0, .s = 0 }, .index = 255 }`
etc.

Change to this in `rgb_backlight.c`:

`.caps_lock_indicator = RGB_BACKLIGHT_CAPS_LOCK_INDICATOR,`
etc.

Then to override in a keymap directory's `config.h`:

`#undef RGB_BACKLIGHT_CAPS_LOCK_INDICATOR 
#define RGB_BACKLIGHT_CAPS_LOCK_INDICATOR { .color = { .h = ?, .s = ? }, .index = ? }`
etc.

I will make the changes tonight and do a PR.

**Comment by stanrc85** [comment]
Thanks! I can add some details to the readme about it.

**Comment by Wilba6582** [comment]
Which readme?

**Comment by stanrc85** [comment]
For hs60/v2

File: keyboards/hs60/v2/keymaps/stanrc85/rules.mk [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # project specific files
 SRC =	keyboards/zeal60/zeal60.c \
-		keyboards/zeal60/rgb_backlight.c \
+    keyboards/zeal60/rgb_backlight.c \
```
Comment (drashna) [inline keyboards/hs60/v2/keymaps/stanrc85/rules.mk]: Could this be completely reverted, so the PR doesn't touch the file? 

```suggestion
		keyboards/zeal60/rgb_backlight.c \
```

**Comment by drashna** [comment]
Thanks!

**Pull request merged**